src/BloomExe/web/EnhancedImageServer.cs

```diff
@@ -541,6 +543,44 @@ private static bool ProcessAnyFileContent(IRequestInfo info, string localPath)
 			return true;
 		}
 
+		private bool ProcessCssFile(IRequestInfo info, string localPath)
+		{
+			if  (CurrentBook == null) return false;
```

Comment:
> How will this affect working on books that aren't the current book

I think we need to get the book folder paths included in any URLs that are
book-specific, and not let the server know what the "Current Book" is.